torch/testing/_internal/common_utils.py

```diff
@@ -314,6 +314,15 @@ def run_tests(argv=UNITTEST_ARGS):
 IS_MACOS = sys.platform == "darwin"
 IS_PPC = platform.machine() == "ppc64le"
 
+def is_avx512_vnni_supported():
+    if sys.platform != 'linux':
+        return False
+    with open("/proc/cpuinfo", encoding="ascii") as f:
+        lines = f.read()
+    return all(word in lines for word in ["avx512vnni"])
```

Comment:
Nit
```suggestion
    return "avx512vnni" in lines
```

Reply:
Thanks a lot! I had forgotten to do that after I removed other flags 😞